src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Organize CockroachDB Cloud Clusters Using Folders
+summary: Learn how to use folders to organize CockroachDB Cloud clusters and manage access to them.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/limited-access.md %}
+
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters using folders. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}](managing-access.html).
+
+## How folders work
+
+By default, clusters are created at the level of the organization. Folders provide additional ways to organize clusters and manage access to them. A folder can contain both clusters and descendent folders.
+
+Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID. A folder can contain a mix of child folders and clusters.
+
+If no parent ID is specified for a cluster, it is created at the level of the organization. Clusters and folders at the organization level have their `parent_id` set to `root`.
+
+{{site.data.alerts.callout_success}}
```

Comment:
I think we can simplify this paragraph to one sentence explaining that a move operation updates the parent ID of a resource, and then also touch on the conditions where the move fails. Also the last sentence isn't entirely accurate, only one resource is updated during any given move operation

Reply:
+1.  Probably no need to mention that it's a lightweight operation, or any impl details.

Conditions where a move could fail include:
* Name collision: destination folder contains a folder with the same name (if moving a folder), or destination folder contains a cluster with the same name (if moving a cluster)  NOTE: a folder CAN have the same name as a cluster - that's allowed
* Sibling limits: destination folder is full (has hit the 200 resource limit)
* Depth limits: moving this folder would cause the destination to exceed the max allowable depth
* Structural limits: certain moves are illegal, such as moving a folder into itself, or moving a folder into one of its children

Reply:
Done